gsoc2018-bharat/src/sparql.py

```diff
@@ -2,20 +2,25 @@
 import asyncio
 from urllib.error import HTTPError
 from SPARQLWrapper.SPARQLExceptions import EndPointInternalError
+import json
 
 sparql = SPARQLWrapper("http://dbpedia.org/sparql")
 final_pool = []
+
 query = """
-    PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+    PREFIX rdfs:<http://www.w3.org/2000/01/rdf-schema#>
+    PREFIX rdf:<http://www.w3.org/1999/02/22-rdf-syntax-ns#>
     PREFIX dbo: <http://dbpedia.org/ontology/>
-    SELECT DISTINCT ?s ?abstract WHERE {
+    SELECT DISTINCT ?s ?abstract ?url ?p
+    WHERE {
         ?s a/rdfs:subClassOf* dbo:Organisation .
```

Comment:
As discussed via call, I suggest to split the query into two queries and merge their results later. Having the abstracts multiplied by the number of classes isn't a good idea.